### PR TITLE
extracts myreports out of dashboard and into its own screen.

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

File: app/components/ilios-navigation.hbs [hunk 1]
```diff
@@ -198,6 +198,23 @@
           </LinkTo>
         </li>
       {{/if}}
+      <li>
```
Comment (jrjohnson) [inline app/components/ilios-navigation.hbs:201]: Feels to me like this should be above admin, personal opinion only.

File: app/router.js [hunk 2]
```diff
@@ -154,4 +154,8 @@ Router.map(function () {
     path: 'data/programyears/:program_year_id/objectives',
   });
   this.route('search');
+  this.route('reports', function () {
+    this.route('subject', { path: '/reports/subject/:report_id' });
```
Comment (jrjohnson) [inline app/router.js:158]:
```suggestion
    this.route('subject', { path: '/reports/subjects/:report_id' });
```
keeping plural URL to match everything else.

**Review by jrjohnson — COMMENTED** [review]
Super early review, direction looks great to me.

File: app/router.js (hunk 2 above)
Comment (stopfstedt) [reply app/router.js]: :heavy_check_mark:

File: app/components/ilios-navigation.hbs (hunk 1 above)
Comment (stopfstedt) [reply app/components/ilios-navigation.hbs]: :heavy_check_mark:

File: config/icons.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 module.exports = function () {
   return {
     'pro-light-svg-icons': ['clipboard-list', 'lightbulb-slash'],
-    'pro-solid-svg-icons': ['lightbulb-on'],
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply config/icons.js]: it was on accident, thx for catching that. fixed now :heavy_check_mark:

**Review by dartajax — COMMENTED** [review]
It looks good to me - I will check my download though. How difficult would it be to add a record counter like on "Courses" so when you filter by year, for example the count would update?

UPDATE: I can file this as a later eventual follow-up ticket though since we don't have this in the current My Reports.

**Comment by stopfstedt** [comment]
> It looks good to me - I will check my download though. How difficult would it be to add a record counter like on "Courses" so when you filter by year, for example the count would update?
> 
> UPDATE: I can file this as a later eventual follow-up ticket though since we don't have this in the current My Reports.

yes, that might best be handled as a follow-up item.

**Review by dartajax — APPROVED** [review]
looks good to me

File: tests/integration/components/reports/subjects-test.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  setupIntl(hooks, 'en-us');
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    this.user = this.server.create('user');
+    const jwtObject = {
+      user_id: this.user.id,
+    };
+    const encodedData = window.btoa('') + '.' + window.btoa(JSON.stringify(jwtObject)) + '.';
+    await authenticateSession({
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply tests/integration/components/reports/subjects-test.js:22]: i'll have a look. this is an old test that got ported over, perhaps we can lose this relic.

File: tests/integration/components/reports/subject-test.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+import { component } from 'ilios/tests/pages/components/reports/subject';
+
+module('Integration | Component | reports/subject', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks, 'en-us');
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    this.user = this.server.create('user');
+    const jwtObject = {
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply tests/integration/components/reports/subject-test.js:17]: same, looking into it...

File: tests/integration/components/reports/subjects-test.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (stopfstedt) [reply tests/integration/components/reports/subjects-test.js]: :heavy_check_mark:

File: tests/integration/components/reports/subject-test.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (stopfstedt) [reply tests/integration/components/reports/subject-test.js]: :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**